b2luigi/batch/processes/lsf.py

```diff
@@ -30,17 +30,23 @@ class LSFProcess(BatchProcess):
     """
     Reference implementation of the batch process for a LSF batch system.
 
-    Additional to the basic batch setup (see :ref:`batch-label`), additional
-    LSF-specific things are:
+    Additional to the basic batch setup (see :ref:`batch-label`), there are
+    LSF-specific settings. These are:
```

Comment:
Here we can make "settings" a link to the settings documentation and then we don't need to specify below which settings interface we mean.

```suggestion
    LSF-specific :meth:`settings <b2luigi.set_setting>`. These are:
```